documentation/rosetta-modelling-component.md

```diff
@@ -1646,14 +1660,15 @@ xField string (1..1);
 A set when mapping is used to set an attribute to a value derived from the input document if a given when clause is met -  e.g. :
 
 ```
-execution Execution (0..1) <"The execution ...">;
-  [synonym CME_SubmissionIRS_1_0 value TrdCaptRpt set when "TrdCaptRpt.VenuTyp" exists]
+alternativeCombustible FuelEnum (1..1)
+  [synonym CONDITIONAL_SET_TO_EXAMPLE_7 set to FuelEnum -> ELECTRICITY when "engineDetail->fuelDetail->alternativeCombustible" exists]
```

Comment:
"set" is missing before the "when"

Reply:
See original example
execution Execution (0..1) <"The execution ...">;
  [synonym CME_SubmissionIRS_1_0 value TrdCaptRpt set when "TrdCaptRpt.VenuTyp" exists]

Reply:
the term "value" is also missing as per the example

Reply:
So I can do 2 things: 
1- Create or modify an existing test of conditional set so in order that this new have set when" + <"path"> + "exists"
2- Modify the original example so it has the same mapping result but with the new taxonomy and do not modify any of the test samples in demo translate
I would suggest to do 1st one, so the examples in documentation refers to the ones on demo-translate but what you think?